content/en/docs/ops/configuration/mesh/config-resource-ready/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Wait for Resource Status to Apply Configuration
+description: Describes how to wait to apply mesh configuration until a resource reaches a given status or readiness.
+weight: 15
+---
+
+{{< warning >}}
+This feature is in the Alpha stage, see
+[Istio Feature Status](/about/feature-stages/). Your feedback is welcome in the
+[Istio User Experience discussion](https://discuss.istio.io/c/UX/23). Currently,
+this feature is tested only for single, low volume clusters with a single
+control plane revision.
+{{< /warning >}}
+
+Istio's mesh configuration is declarative, which means that you define a
+configuration and Istio propagates the changes through the mesh over time. As a
+result, Istio might attempt to apply consecutive configuration changes before
+the target resources are ready.
```

Comment:
`your code might attempt to use your service mesh before the relevant resources are ready.`

Reply:
fixed